curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-intermediate-algorithmic-thinking-by-building-a-dice-game/64ba11fef6188b0261213803.md

```diff
@@ -0,0 +1,452 @@
+---
+id: 64ba11fef6188b0261213803
+title: Step 67
+challengeType: 0
+dashedName: step-67
+---
+
+# --description--
+
+Again, You should create an `if` statement with the condition `count >= 4 && count > highestCount`. This condition checks if the value of count is greater than or equal to 4 and if it's greater than the current `highestCount`. Inside the `if` statement, set `highestCount` to the value of `count`. This will update `highestCount` if the current `count` is the highest one encountered so far that is also greater than or equal to 4. The main purpose of this is to detect a four of a kind.
```

Comment:
This step might be obsolete. As it stands condition added in previous step will apply to `count >= 4` as well.